packages/edit-site/src/components/dataviews/reset-filters.js

```diff
@@ -1,26 +1,26 @@
 /**
  * WordPress dependencies
  */
-import { BaseControl, Button } from '@wordpress/components';
+import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
 export default ( { view, onChangeView } ) => {
 	return (
-		<BaseControl>
-			<Button
-				disabled={ view.search === '' && view.filters?.length === 0 }
-				variant="tertiary"
-				onClick={ () => {
-					onChangeView( ( currentView ) => ( {
-						...currentView,
-						page: 1,
-						search: '',
-						filters: [],
-					} ) );
-				} }
-			>
-				{ __( 'Reset filters' ) }
-			</Button>
-		</BaseControl>
+		<Button
+			disabled={ view.search === '' && view.filters?.length === 0 }
+			__experimentalIsFocusable={ true }
+			__next40pxDefaultSize={ true }
```

Comment:
Nit `={ true }` can be dropped from all the boolean props.

Reply:
https://github.com/WordPress/gutenberg/pull/55992/commits/7906c6f9322aff78b9c9df15c7c9ca77ee4354f8